Analyticsv3Sample/README.md

```diff
@@ -0,0 +1,50 @@
+
+## DAIMTO Google Analytics API Sample for .NET  ##
+
+View and manage your Google Analytics data
+
+## Description ##
+The DAIMTO Google Analytics API Sample for .NET instructs you in the usage of the Google APIs client library for .NET.
+
+The sample contains OAuth2.0 authentication, service account, and and public API.   As well as methods for each of the Resources avaliable in the Analytics API.  
```

Comment:
Consider adding space between OAuth to 2.0

Maybe also rephrase it to:
The sample contains usage of the Google Analytics V3 using OAuth2.0 and Service Account.